key/key.go

```diff
@@ -59,6 +60,39 @@ func (op *Op) applyOpts(opts []OpOption) {
 	}
 }
 
+func WithTime(t uint64) OpOption {
+	return func(op *Op) {
+		op.time = t
+	}
+}
+
+func WithTargetAmount(ta uint64) OpOption {
+	return func(op *Op) {
+		op.targetAmount = ta
+	}
+}
+
+// To deduct transfer fee from total spend (output).
+// e.g., "units.MilliAvax" for X/P-Chain transfer.
+func WithFeeDeduct(fee uint64) OpOption {
+	return func(op *Op) {
+		op.feeDeduct = fee
+	}
+}
+
+func GetHRP(networkID uint32) string {
```

Comment:
I see this in avago. Any benefit to just keeping a copy here? And if a copy here is preferrable, let's document this with a similar comment in avago:

// GetHRP returns the Human-Readable-Part of bech32 addresses for a networkID

Reply:
file is removed